### DISCUSSION TITLE
suppervisor error amqp killed in context shut_down

### DISCUSSION THREAD

**Discussion by lliiang** [rabbitmq/rabbitmq-server#3597] [Ideas]
![1bd769e53680d5b3c78499830c694a7](https://user-images.githubusercontent.com/92851689/138071483-4ed397d9-ae03-4ed8-a4da-571b0f5ed1f1.jpg)

**Comment by lhoguin** [comment]
Please ignore this error it will be fixed when you upgrade to the latest OTP version with the most recent patch release. It is just noise.
